src/logger.c

```diff
@@ -11,7 +11,7 @@
 LoggerData g_logger;
 
 #ifndef min
-#define	min(a,b)	(((a) < (b)) ? (a) : (b))
+#define min(a,b) (((a) < (b)) ? (a) : (b))
```

Comment:
Ah, I see it's actually now defined in two places, surely it should go in a header and be included now?